examples/create-react-app-with-typescript/package.json

```diff
@@ -6,7 +6,8 @@
     "material-ui": "next",
     "react": "latest",
     "react-dom": "latest",
-    "react-scripts-ts": "latest"
+    "react-scripts-ts": "latest",
+    "typescript": "latest"
```

Comment:
This should technically go in `devDependencies` (and `@types/react`, `@types/react-dom` should be in `dependencies`).